### PR TITLE
Feature/scoreboard html

### PR DESCRIPTION
**Pull request by Tthomas63**

### CODE REVIEW

**Comment by Tthomas63** [comment]
This is looking good!

- What is the advantage of the .min files over CDN? (I'm assuming speed?)
- Need to merge a few things together so we can add more info to this.

**Comment by Tthomas63** [comment]
@ckcollab Good to resolve conflicts and merge?

File: content/html/riotjs/scoreboard.tag [hunk 1]
```diff
@@ -46,12 +50,20 @@
...
-                team_list: JSON.parse(player.getAll()),
-            })
-            console.log(self.team_list)
-            window.setTimeout(update_loop, 1000);
+            // player may not exist if we're testing in local browser, wait for it
+            if (typeof player !== 'undefined') {
+                self.update({
+                    teams: JSON.parse(player.getAll()),
+                })
+                console.log(self.teams)
```
Comment (ckcollab) [inline content/html/riotjs/scoreboard.tag]: do we wanna pull stuff like this or leave it in?

Comment (Tthomas63) [reply content/html/riotjs/scoreboard.tag]: Pull the console.log probably, I believe the latest should still be receiving the data

**Pull request merged**